README.rst

```diff
@@ -174,37 +172,33 @@ The rules which are available by default:
 * `booking-refuse` (Invia un'email all'utente quando la prenotazione è stata rifiutata)
 * `booking-confirm` (Conferma automatica prenotazioni)
 
-
-You can create your own email templates for the booking events(confirm, refuse, create, delete).
-The temlates are being saved in the PrenotazioniFolder object.
+You can also use some placeholders that will be replaced with some booking infos. Here is a list:
 
 The template variables list:
 
-* ``${title}`` - booking title.
-* ``${booking_gate}`` - booking gate.
-* ``${booking_human_readable_start}`` - booking human readable start.
-* ``${booking_date}`` - booking date.
-* ``${booking_end_date}`` - booking end date.
-* ``${booking_time}`` - booking time.
-* ``${booking_time_end}`` - booking time end.
-* ``${booking_code}`` - booking code.
-* ``${booking_type}`` - booking type.
-* ``${booking_print_url}`` - booking print url.
-* ``${booking_url_with_delete_token}`` - booking url with delete token.
-* ``${booking_user_phone}`` - booking user phone.
-* ``${booking_user_email}`` - booking user email.
-* ``${booking_user_details}`` - booking user details.
-* ``${booking_office_contact_phone}`` - booking office contact phone.
-* ``${booking_office_contact_pec}`` - booking office contact pec.
-* ``${booking_office_contact_fax}`` - booking office contact fax.
-* ``${booking_how_to_get_to_office}`` - booking how to get to office.
-* ``${booking_office_complete_address}`` - booking office complete address.
-* ``${booking_user_details}`` - booking user details
-* ``${booking_requirements}`` - booking requeirements.
-* ``${prenotazioni_folder_title}`` - prenotazioni folder title.
-* ``${booking_user_details}`` - booking details.
+* ``${title}`` - Booking title.
+* ``${booking_gate}`` - Booking gate.
+* ``${booking_human_readable_start}`` - Booking human readable start datetime.
+* ``${booking_date}`` - Booking date.
+* ``${booking_end_date}`` - Booking end date.
+* ``${booking_time}`` - Booking time.
+* ``${booking_time_end}`` - Booking time end.
+* ``${booking_code}`` - Booking code.
+* ``${booking_type}`` - Booking type.
+* ``${booking_print_url}`` - Booking summary url.
+* ``${booking_url_with_delete_token}`` - Booking url to detele page.
+* ``${booking_user_phone}`` - Booking user phone.
+* ``${booking_user_email}`` - Booking user email.
+* ``${booking_user_details}`` - Booking user details.
+* ``${booking_office_contact_phone}`` - Booking office contact phone.
```

Comment:
Are those variable here or in design.plone.ioprenoto? Because these are related to iocomune and not with plain redturtle.prenotazioni

Reply:
The are redturtle.prenotazioni defined fields https://github.com/RedTurtle/redturtle.prenotazioni/blob/fe34c21033a25352ae2cfd2484684f2e864e451c/src/redturtle/prenotazioni/content/prenotazione.py#L135C1-L154C6

Reply:
i'm talking about office fields